Content.Server/Administration/Commands/LoadGameMapCommand.cs

```diff
@@ -46,13 +46,15 @@ public void Execute(IConsoleShell shell, string argStr, string[] args)
                         return;
                     }
 
-                    if (args.Length is 4 && int.TryParse(args[2], out var x) && int.TryParse(args[3], out var y))
+                    if (args.Length == 4 && int.TryParse(args[2], out var x) && int.TryParse(args[3], out var y))
```

Comment:
the == here should be equal or greater right?

Reply:
...yep i'm tired